mlrun/api/crud/pipelines.py

```diff
@@ -99,6 +99,57 @@ def list_pipelines(
 
         return total_size, next_page_token, runs
 
+    @staticmethod
+    def _get_project_from_experiment_name(name: str) -> str:
+        try:
+            return name.partition("-")[0]
```

Comment:
how will it behave for projects named "a-b"?

Reply:
Ops, that was naive of me, sorry 😂.

I'm not sure on how to filter experiments by projects though. Maybe iterating over all project workflows as well and combining that with the project name would be the safest option?